desktop/apps/auction2/components/DOM.js

```diff
@@ -31,14 +31,14 @@ export default class DOM extends Component {
   addEventListeners () {
     this.$body = this.$('body')
     this.$body.on('click', '.artsy-checkbox', scrollToTop)
-    this.$(window).on('scroll.auction-page-artworks', infiniteScroll(this.props.store))
+    this.$(window).on('scroll.auction2-page-artworks', infiniteScroll(this.props.store))
```

Comment:
Is this necessary because the CSS for both `/auction` and `/auction2` are being delivered?